examples_tests/50.RadixSort/Histogram.comp

```diff
@@ -0,0 +1,62 @@
+#version 450
+
+#define VT (1 << 2)
+#define WG_DIM (1 << 9)
```

Comment:
there's a define we use by convention `_NBL_GLSL_WORKGROUP_SIZE_` in all the other examples (you still gotta define it)

Reply:
yeah, just another inconsistency I'll get rid of at the end